### DISCUSSION TITLE
Unable to remove nx. the .nx folder keeps coming back

### DISCUSSION THREAD

**Discussion by OlivierPerceboisGarve** [nrwl/nx#21176] [General]
Hi
I have a project that I do not touch often and that is not supposed to use nx.
There is a .nx folder that appears after npm i.
I am unsuccessful at removing it, it keeps coming back. I tried to :
- delete the .nx folder
- delete package-lock
- empty npm cache with "npm cache clean --force"
- search the project for occurences of "@nx" and "@nrwl" ---> no occurences

what else can I try to make it go away ? what can be the thing that makes it coming back ? Some kind of hook to npm i ?

thanks for any help

**Comment by bougwal** [comment]
Hello, you can read about the original report here: [ Nx commands have suddendly started to create the .nx folder on the project root folder instead node_modules on version 16.10 ](https://github.com/nrwl/nx-console/issues/1975). Read the full PR discussion and in the bottom you will find it closed as done after a fix has been merged.

Hope that helps

**Reply by OlivierPerceboisGarve** [reply]
thanks a lot for your help. This explains why it appeared suddenly. 

Apparently my install of nx is somehow global but I do not know how to update or remove it.

**Comment by marfrede** [comment]
for me it was the VSCode Extension `Nx Console` (nrwl.angular-console) causing this.\
It was running a process in the background. After uninstalling the extension and deleting the `.nx`-folder, it never came back.

**Reply by iuriiDidenko** [reply]
This is a live-saver, thanks!
